CoreInteractiveGraph/src/au/gov/asd/tac/constellation/graph/interaction/plugins/io/SaveAsAction.java

```diff
@@ -189,9 +192,8 @@ public void actionPerformed(final ActionEvent e) {
 
                     // Take a screenshot in a separate thread in parallel.
                     new Thread(() -> RecentGraphScreenshotUtilities.takeScreenshot(selectedFile.getName()), "Take Graph Screenshot").start();
-                } catch (IOException ex) {
-                    Exceptions.printStackTrace(ex);
-                    Exceptions.attachLocalizedMessage(ex, Bundle.MSG_SaveAsFailed(selectedFile.getName(), ex.getLocalizedMessage()));
+                } catch (final IOException ex) {
+                    LOGGER.log(Level.WARNING, Bundle.MSG_SaveAsFailed(selectedFile.getName(), ex.getLocalizedMessage()));
```

Comment:
I think I would also make the level `SEVERE` here. If you're also able to pass the exception as a parameter here that would be great